fury/primitive.py

```diff
@@ -328,3 +331,77 @@ def _fexp(x, p):
     vertices = np.ascontiguousarray(xyz)
 
     return vertices, sphere_triangles
+
+
+def prim_tetrahedron():
+    """
+    returns vertices and triangles to a tetrahedron
+    that has a side length of two units.
+
+
+    Returns
+    -------
+    pyramid_vert: numpy.ndarray array
+        4 vertices coordinates
+    triangles: numpy.ndarray
+        4 triangles representing the tetrahedron
+    """
+    pyramid_vert = np.array([[0.0, 0.0, 0.0], [2.0, 0.0, 0.0], [1.0, math.sqrt(
```

Comment:
Can you avoid to put everything in one line? Try to follow the same format as `pyramid_triag`